### PR TITLE
Implement Renovate

### PR DESCRIPTION
**Pull request by hcsa73**

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "extends": [
+        "config:recommended",
+        ":dependencyDashboard"
+    ],
+    "enabledManagers": [
+        "gomod",
+        "github-actions"
+    ],
+    "gomod": {
+        "packageRules": [
```
Comment (vicentepinto98) [inline .github/renovate.json:12]: what do these pacakgeRules do?

File: .github/workflows/renovate.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    name: Renovate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v39.0.5
+        with:
+          configurationFile: .github/renovate.json
+          token: ${{ secrets.RENOVATE_TOKEN }}
```
Comment (vicentepinto98) [inline .github/workflows/renovate.yaml:19]: which token did you use?

**Review by vicentepinto98 — APPROVED** [review]

File: .github/renovate.json (hunk 1 above)
Comment (hcsa73) [reply .github/renovate.json:12]: Group dependency updates, so that it doesn't create a PR per go.mod

File: .github/workflows/renovate.yaml (hunk 2 above)
Comment (hcsa73) [reply .github/workflows/renovate.yaml:19]: Github's PAT from my account. Will replace it with PAT from the technical account

**Pull request merged**